src/features/invites/components/InvitesList/InvitesList.tsx

```diff
@@ -76,12 +87,7 @@ const InvitesList = () => {
         </AccordionSummary>
         <AccordionDetails>
           {invites.map((invite) => (
-            <InviteListItem
-              loading={loading}
-              acceptInvite={() => acceptInvite(invite.channelAddress)}
-              key={invite.invitationId}
-              invite={invite}
-            />
+            <InviteListItem key={invite.invitationId} acceptInvite={acceptInvite} invite={invite} />
           ))}
           {invites.length === 0 && (
```

Comment:
Could look better this way
```suggestion
          {!invites.length && (
```